app/file.php

```diff
@@ -1,2 +1,50 @@
 <?php
 
+declare(strict_types = 1);
+$root = dirname(__DIR__) . DIRECTORY_SEPARATOR;
+define('FILES_PATH', $root . 'files' . DIRECTORY_SEPARATOR);
+
+class File
+{
+    private string $file_name;
+    private string $file_path;
+    private $raw_data;
```

Comment:
Which type is $raw_data?

Reply:
Resource but this variable cannot be type hinted.